torchaudio/functional/functional.py

```diff
@@ -626,9 +646,11 @@ def phase_vocoder(
     complex_specgrams_stretch = torch.stack([real_stretch, imag_stretch], dim=-1)
```

Comment:
@anjali411 Here, we are constructing complex tensor from magnitude and angle. Is there a more elegant way to do this?

Reply:
yeah `torch.polar(abs, angle)`